APIClient/Program.cs

```diff
@@ -11,14 +11,21 @@ class Program
     {
         static async Task Main()
         {
-            var weatherForecast = new WeatherForecastApi("https://localhost:7016/");
+            var weatherForecastApi = new WeatherForecastApi("https://localhost:7016/");
```

Comment:
these are called "magic strings". Strings (and numbers) that are placed inside the code. Usually you want to avoid them. If you need to pass some kind of value do **one** of the following:

- Add a configuration file
- Put the string in a constant variable at the beginning of method or class